pkg/dynamic/dynamic.go

```diff
@@ -0,0 +1,18 @@
+package dynamic
+
+import (
+	"k8s.io/client-go/dynamic"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+)
+
+func CreateClientSet() (*dynamic.Interface, error){
```

Comment:
Why are returning `*dynamic.Interface` rather than `*dynamic.ClientSet`
No need to fix in this PR, will raise an issue for this.

Reply:
Raised another Pr for this